### PR TITLE
made android maps toolbar controllable by prop

### PR DESCRIPTION
**Pull request by heydabop**
Resolves [#158](https://github.com/lelandrichardson/react-native-maps/issues/158)

### CODE REVIEW

File: components/MapView.js [hunk 1]
```diff
@@ -132,6 +132,14 @@ var MapView = React.createClass({
     pitchEnabled: PropTypes.bool,
 
     /**
+     * `Android only`
```
Comment (christopherdro) [inline components/MapView.js]: nit: remove this line and add `@platform android` at the end of the prop description.

**Comment by christopherdro** [comment]
Sweet! Outside of the small change it looks good to me.

**Comment by heydabop** [comment]
@christopherdro Ah, not sure how I missed all the other parts doing that. Fixed it!

**Comment by christopherdro** [comment]
@felipecsl I know you just did a release to maven before this PR. Is it easy to update the the build on maven to include this?

**Comment by felipecsl** [comment]
@christopherdro unlike NPM, Maven artifacts are immutable. Once you upload it you can't modify it (I believe you cant even delete it). So no, we'd need a new release.

**Comment by christopherdro** [comment]
@felipecsl Ok. Does it make sense to try and plan out a steady release schedule? Say every 2-3 weeks? If we release a new version that doesn't have any native code changes, would we still be required to release a new version to maven that matches?

**Comment by felipecsl** [comment]
sounds good to me! I believe we should try to keep all versions in sync even if there are no changes. It would be pretty confusing if the npm package is at version 0.9.0 but the latest Android jar is 0.6.0

**Comment by christopherdro** [comment]
Sounds like a plan!

**Pull request merged**